checkov/terraform/checks/graph_checks/aws/ElasticSearchDedicatedMasterEnabled.yaml

```
@@ -0,0 +1,16 @@
+metadata:
+  id: "CKV2_AWS_59"
+  name: "Ensure Elastic Search has dedicated master node enabled"
+  category: "GENERAL_SECURITY"
+definition:
+  and:
+    - cond_type: attribute
+      resource_types: "aws_elasticsearch_domain"
+      attribute: "cluster_config.dedicated_master_enabled"
+      operator: "jsonpath_equals"
```

```suggestion
      operator: "equals"
```
no need for `jsonpath` here 🙂

---

Noted 🙂